scripts/prepare-release.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+const shell = require('shelljs');
+
+// Grab provided args. Possible options equal patch, minor, major.
+const [,, ...args] = process.argv
+
+let version = args[0];
+
+// check for system dependencies
+if (!shell.which('git')) {
+  shell.echo('Sorry, this script requires git');
+  shell.exit(1);
+}
+
+// if no version, set minor as default version bump
+if(!version) {
+	version = 'minor'
+}
+
+// bump version & tag
+console.log(version);
+shell.exec(`npm version ${version} -m 'Release %s' `)
+
+// push up to github, assumes origin is where you want to push
+shell.exec(`git push origin && git push --tags`)
```

Comment:
Instead of assuming `origin` as the remote name, can we parse the correct remote name out based on URL? Something like:

```sh
for remote in $(git remote);
do
    if [[ $(git remote get-url $remote) == *"adobe/spectrum-css"* ]]; then
        echo "the adobe github remote is under the $remote name";
    fi
done
```

... yields the following in my repo w/ github.com/adobe/spectrum-css under 'origin':

> the adobe github remote is under the origin name

Reply:
Oh dang this will push all the tags, so if we have any local/wrong tags, they'll go too. We should only push the tag we created for this release.

Reply:
Fixed to only push the release tag now

Reply:
It will also figure out the correct remote name to push to (finds the one that has `adobe/spectrum-css` in it.